frontend/src/components/OperationStepForm/OperationStepForm.tsx

```diff
@@ -37,6 +37,7 @@ const schema = Yup.object().shape({
 const queries = [
   { key: 'select', icon: 'check', text: 'Select', value: 'select' },
   { key: 'filter', icon: 'filter', text: 'Filter', value: 'filter' },
+  { key: 'text_filter', icon: 'filter', text: 'Text Filter', value: 'text_filter' },
```

Comment:
Could you please rename it as `Advanced Filter` ... this make it seem like it's filtering text ... rename in all other occurrences as well.

Reply:
Updated